sdk/src/main/java/com/podio/sdk/androidasynchttp/AndroidAsyncHttpRequest.java

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
I know that this is all over the place, 
but I would remove the Copyright note. This is 90's practice. Now you put in the `README.md` if you want and that's it.